studies/tasks.py

```diff
@@ -410,3 +412,12 @@ def build_zipfile_of_videos(
         from_email=settings.EMAIL_FROM_ADDRESS,
         **context,
     )
+
+
+@app.task(bind=True)
+def delete_video_from_cloud(task, s3_video_name):
+    """Delete videos in S3.
+
+    Meant to have a delay of about 7 days.
+    """
+    S3_RESOURCE.Object(settings.BUCKET_NAME, s3_video_name).delete()
```

Comment:
What happens if the video isn't found / was already deleted? Is that error a problem for anything?

Reply:
Also, what (if anything) is the process for cancelling this task? (We should write some description of how this works for lookit-docs, both for researchers to understand in what sense the data is gone / in what edge cases they might briefly have access to it, and for us to reference in case anyone ever says "I didn't mean to withdraw my data!")

Reply:
it'll just bork the celery task, which is nbd. I can add logging if you want but honestly the error raised by the boto3 client will be just as informative as anything I can provide

Reply:
Actually looks like boto3 might not even raise an error? But either way, should be fine.